GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/rawfile.cpp

```diff
@@ -438,32 +433,18 @@ int RawFileClass::Open(int rights)
 				break;
 
 			case READ:
-				#ifdef _UNIX
-					Handle = fopen(Filename, "r");
-				#else
-					Handle = CreateFileA(Filename, GENERIC_READ, FILE_SHARE_READ,
-												NULL, OPEN_EXISTING, FILE_ATTRIBUTE_NORMAL, NULL);
-				#endif
+				Handle = fopen(Filename, "rb");
 				break;
 
 			case WRITE:
-				#ifdef _UNIX
-					Handle = fopen(Filename, "w");
-				#else
-					Handle = CreateFileA(Filename, GENERIC_WRITE, 0,
-												NULL, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
-				#endif
+				Handle = fopen(Filename, "wb");
 				break;
 
 			case READ|WRITE:
-				#ifdef _UNIX
-					Handle = fopen(Filename, "w");
-				#else
-					// SKB 5/13/99 use OPEN_ALWAYS instead of CREATE_ALWAYS so that files
-					//					does not get destroyed.
-					Handle = CreateFileA(Filename, GENERIC_READ | GENERIC_WRITE, 0,
-												NULL, OPEN_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
-				#endif
+				Handle = fopen(Filename, "rb+");
```

Comment:
Preserve a comment to describe the desired behaviour here, otherwise the next reader may be confused about the 2 step fopen.